assets/sharedClientFunctions.js

```diff
@@ -47,7 +48,7 @@ module.exports = {
         return "Make sure you enter all the information";
       }
       if (((ref.type === "giveaway" || ref.type === "eggcheck") && !regexpGive.test(url)) ||
-        (ref.type !== "giveaway" && ref.type !== "misc" && ref.type !== "eggcheck" && !regexp.test(url)) ||
+        (ref.type !== "giveaway" && ref.type !== "misc" && ref.type !== "eggcheck" && (!regexp.test(url) && !regexpMobile.test(url))) ||
         (ref.type === "misc" && !regexpMisc.test(url))) {
```

Comment:
Same here

Reply:
The MISC regex is super generous and only checks for reddit.com so we are covered on MISC